tests/foreman/ui_airgun/test_container.py

```diff
@@ -0,0 +1,70 @@
+from nailgun import entities
+
+from robottelo.constants import ENVIRONMENT, DEFAULT_CV, DOCKER_REGISTRY_HUB, REPO_TYPE
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture
+from robottelo.config import settings
+from robottelo.decorators import (
+    tier2,
+    upgrade,
+)
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def create_docker(module_org):
+    docker_url = settings.docker.external_url
+    return entities.DockerComputeResource(
+        name=gen_string('alpha'),
+        organization=[module_org],
+        url=docker_url,
+    ).create()
+
+
+@fixture(scope='module')
+def module_prod(module_org):
+    return entities.Product(organization=module_org).create()
+
+
+@tier2
+@upgrade
+def test_positive_create_with_compresource(session, create_docker, module_prod):
+    """Create containers for a compute resource
+
+    :id: 299fc13d-5c2f-4642-a6de-735c89fdd096
+
+    :expectedresults: The docker container is created for the compute
+        resource
+
+    :CaseLevel: Integration
+    """
+    deploy_on = create_docker.name + ' (Docker)'
+    lifecycle_environment = ENVIRONMENT
+    content_view = DEFAULT_CV
```

Comment:
Pls avoid creating single-use variables and just use those constants directly.
Moving variables to the top of test makes sense when there's a chance you'll change those values. But that's not the case here - as your test doesn't create lce or cv you can't change these vars without updating rest of the test (unlike, let's say, `upstream_repository_name`).